core/scheduler/build.gradle

```diff
@@ -66,6 +66,9 @@ dependencies {
         resolutionStrategy.force "com.typesafe.akka:akka-http-spray-json_${gradle.scala.depVersion}:${gradle.akka_http.version}"
         resolutionStrategy.force "com.typesafe.akka:akka-parsing_${gradle.scala.depVersion}:${gradle.akka_http.version}"
         resolutionStrategy.force "com.typesafe.akka:akka-http_${gradle.scala.depVersion}:${gradle.akka_http.version}"
+        resolutionStrategy.dependencySubstitution {
```

Comment:
I'm curious why do we use the `dependencySubstitution` approach here instead of the `force` approach used by all the other versions above?

Reply:
Originally I was using the `force` approach, which also works fine, but then I thought that this approach looks more "idiomatic". 
I can put back the `force` if you think that it adds less cognitive load.